README.md

```diff
@@ -90,6 +90,15 @@ Here's a list of live apps made with the Uno Platform for WebAssembly.
 
 Let us know if you've made your app publicly available, we'll list it here!
 
+# Builds
+
+Build process summaries :
+
+* Uno.UI - CI (Master)  
+[![Uno.UI - CI (Master) status](https://uno-platform.visualstudio.com/Uno%20Platform/_apis/build/status/Uno%20Platform/Uno.UI%20-%20CI?branchName=master)](https://uno-platform.visualstudio.com/Uno%20Platform/_build?definitionId=5)
+* Uno.UI - CI (Stable)  
+[![Uno.UI - CI (Stable) status](https://uno-platform.visualstudio.com/Uno%20Platform/_apis/build/status/Uno%20Platform/Uno.UI%20-%20CI?branchName=release/stable/*)](https://uno-platform.visualstudio.com/Uno%20Platform/_build?definitionId=5)
```

Comment:
@ghuntley / @MatFillion
We cannot use "release/stable/*" as a branch name for the status badges.
It would be painful in my opinion the need to update the branch name here for any new release/stable branch.
So let me know if you are thinking of a better idea for that part.

Reply:
Thanks for the identifiers - going to refactor your PR using this info and combine with shields.io